src/resourcesSpec.ts

```diff
@@ -173,8 +389,6 @@ export function getItemType(baseType: string, propType: string, key: string) {
 
     if (!property.ItemType) {
         return undefined;
-    } else if (property.ItemType === 'Tag') {
```

Comment:
Are there tests covering the 'Tag' type? I seem to remember it not being specified correctly in the resource spec from aws.

Reply:
Yes, the way the `Tag` property type is defined in the AWS Resource Specification is inconsistent with the rest.

If I remember correctly, I decided to remove the artificial way of looking up `Tag` property-types in favor of augmenting the specification at runtime by adding a `ResourceType.Tag` property type for each of the defined resources, this approach seemed more consistent / cleaner to me and also helped to avoid an endless loop that was occuring when `isParameterizedTypeFormat` was being called from somewhere (I can't recall now).